webxr/openxr/mod.rs

```diff
@@ -447,6 +448,7 @@ impl DeviceAPI<Surface> for OpenXrDevice {
             transform,
             inputs: vec![right_input_frame, left_input_frame],
             events,
+            time_ns
```

Comment:
Shouldn't this come from https://docs.rs/openxr/0.10.0/openxr/struct.FrameState.html#structfield.predicted_display_time instead?

Reply:
That's the predicted display time, not the time for the actual frame data.

That said the frame data may itself be predicted. I'm not sure. There are WPT tests ensuring that Performance.now() is greater than the frame timestamp.

Reply:
Basically, I'm not sure what to do here and need spec clarification